src/Common/ErrorCodes.cpp

```diff
@@ -553,6 +553,8 @@
     M(583, ILLEGAL_PROJECTION) \
     M(584, PROJECTION_NOT_USED) \
     \
+    M(996, OPERATION_NOT_PERMITTED) \
```

Comment:
What's the difference from `PATH_ACCESS_DENIED`? And `OPERATION_NOT_PERMITTED` is too general